test/test_mlflow.py

```diff
@@ -0,0 +1,65 @@
+import os
+import pytest
+import mlflow
+
+from azureml.core import Workspace, Experiment, Run
+
+from common_utils import create_sklearn_random_forest_classifier
+
+from interpret_community.mlflow.mlflow import _log_explanation
+from interpret_community.explanation.explanation import load_explanation
+from constants import owner_email_tools_and_ux, DatasetConstants
+from test_serialize_explanation import _assert_explanation_equivalence
+
+
+@pytest.mark.owner(email=owner_email_tools_and_ux)
+@pytest.mark.usefixtures('clean_dir')
+class TestMlflow(object):
+
+    def test_working(self):
+        assert True
+
+    def test_basic_upload(self, iris, tabular_explainer):
+        x_train = iris[DatasetConstants.X_TRAIN]
+        x_test = iris[DatasetConstants.X_TEST]
+        y_train = iris[DatasetConstants.Y_TRAIN]
+
+        model = create_sklearn_random_forest_classifier(x_train, y_train)
+
+        explainer = tabular_explainer(model, x_train)
+        global_explanation = explainer.explain_global(x_test)
+        mlflow.set_experiment('test_experiment')
+        client = mlflow.tracking.MlflowClient()
+        with mlflow.start_run() as run:
+            _log_explanation('test_explanation', global_explanation)
+            artifacts = client.list_artifacts(run.info.run_uuid)
+            os.mkdir('test_explanation')
+            download_path = client.download_artifacts(run.info.run_uuid, '', dst_path='test_explanation')
+        downloaded_explanation = load_explanation(download_path)
+        _assert_explanation_equivalence(global_explanation, downloaded_explanation)
+
+    def test_upload_to_azure(self, iris, tabular_explainer):
+        ws = Workspace.from_config()
```

Comment:
where is this Workspace.from_config coming from?  Why is azure mentioned in interpret-community?

Reply:
This is the world of working with MLflow! We could also put a test in here showing that our files can be moved around with AWS or GCP. You raise a good point though - when this becomes a public feature we'll need a lot more documentation around it.

Reply:
This test can be written in azureml-interpret though, we really don't have to have it here.  It would also make this repository cleaner with a "separation of concerns".  Regardless, the test won't work anyway since we don't have access to an azure workspace in our devops.

Reply:
Yup, this has been changed to a local sqlite database